go-controller/pkg/node/gateway_localnet_linux_test.go

```diff
@@ -41,7 +41,7 @@ func initFakeNodePortWatcher(fakeOvnNode *FakeOVNNode, iptV4, iptV6 util.IPTable
 
 	fNPW := localPortWatcherData{
 		recorder:     fakeOvnNode.recorder,
-		gatewayIP:    v4localnetGatewayIP,
+		gatewayIPs:   map[bool]string{false: v4localnetGatewayIP},
```

Comment:
consider: adding a new test for v6

Reply:
i had  written some tests for v6 and dual stack.  I can bring them in once this PR is merged.